packages/tracing/src/Tracer.ts

```diff
@@ -255,7 +255,7 @@ class Tracer implements TracerInterface {
    * @returns service - Instrumented AWS SDK v2 client
    */
   public captureAWSClient<T>(service: T): T {
-    if (this.isTracingEnabled() === false) return service;
+    if (!this.isTracingEnabled()) return aws;
```

Comment:
This return should be `service`, not `aws`:

```suggestion
    if (!this.isTracingEnabled()) return service;
```